README.md

```diff
@@ -1 +1,42 @@
-# mv_simple_react_calculator
+# Building a simple calculator APP with React
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```

Comment:
It would be nice to add the `Heroku live link` in here 👍

Reply:
Added live version link to readme as requested.